hypothesis_test.ipynb

```diff
@@ -4,7 +4,7 @@
   "metadata": {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #42.          'predicted_fraud': [0.050020, 0.520953, 0.004640, 0.284436, 0.067667,
</pre><p>Are these p-values? If not, what does this field represent?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf_common/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
p-values.